pkgs/top-level/linux-kernels.nix

```diff
@@ -169,6 +169,14 @@ in {
       ];
     };
 
+    linux_rt_6_1 = callPackage ../os-specific/linux/kernel/linux-rt-6.1.nix {
+      kernelPatches = [
+        kernelPatches.bridge_stp_helper
+        kernelPatches.request_key_helper
+        kernelPatches.export-rt-sched-migrate
```

Comment:
Please add kernelPatches.fix-em-ice-bonding as in linux_6_1

Reply:
I thought that was intentional - none of the RT kernels have that patch.
I'll add it.